File: src/ios/FabricPlugin.m
//
//  FabricPlugin.m
//
//

#import <Foundation/Foundation.h>
#import <Cordova/CDV.h>
#import <Fabric/Fabric.h>
#import <Crashlytics/Crashlytics.h>
#import <Crashlytics/Answers.h>

@interface FabricPlugin : CDVPlugin
@end

@implementation FabricPlugin

#pragma mark - Plugin Initialization

- (void)pluginInitialize
{
    [Fabric with:@[[Crashlytics class], [Answers class]]];
}

@end

File: src/android/FabricPlugin.java
package com.sarriaroman.fabric;

import org.apache.cordova.CordovaPlugin;

import com.crashlytics.android.Crashlytics;
import com.crashlytics.android.answers.Answers;

import io.fabric.sdk.android.Fabric;

public class FabricPlugin extends CordovaPlugin {

	@Override
	protected void pluginInitialize() {
		Fabric.with(this.cordova.getActivity().getApplicationContext(), new Crashlytics(), new Answers());
	}

}
